site/en/blog/lighthouse-9-0/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's new in Lighthouse 9.0
+authors:
+  - brendankenny
+date: 2021-11-12
+description: >
+  Lighthouse 9.0 arrives with a refreshed report and a preview of support for user flows.
+hero: 'image/MtjnObpuceYe3ijODN3a79WrxLU2/hcf9uCrSxA4VNMiXHsVn.png'
+alt: 'A new Lighthouse 9.0 report'
+tags:
+  - new-in-lighthouse
+  - lighthouse
+  - chrome-98
+---
+
+Lighthouse is a website auditing tool that helps developers with opportunities and diagnostics to improve the user experience of their sites.
+
+Lighthouse 9.0 is available immediately on the command line and in [Chrome Canary](https://www.google.com/chrome/canary/).
+It will land in Chrome stable in Chrome 98 and be available in [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) within a week.
+
+## API changes
+
+Most users should not run into any workflow-breaking changes with this release. If you run custom Lighthouse audits or use tools that depend on details deep in the Lighthouse report JSON, there may be some breaking changes in 9.0 that you need to be aware of.
```

Comment:
Should we mention changes to the config with performance `diagnostics` and `load-opportunities`?

Reply:
Elizabeth and I at least have talked about keeping programmatic user breaking changes in the changelog and letting this be more general user changes (like someone who generally only uses PSI or devtools), but others feel free to weigh in on that if you think we should include it.